### DISCUSSION TITLE
Frozen group column do not freeze

### DISCUSSION THREAD

**Discussion by lionep** [Comcast/react-data-grid#3475] [Q&A]
Hey !

I'm trying to have some frozen columns with group, and some unfrozen columns also with groups.  
The issue I encounter is that the frozen column group do not freeze, see video attached.

Is this possible to manage ?

Thanks !

https://github.com/adazzle/react-data-grid/assets/968319/49d68152-58ec-4dee-9234-291abf2582f7

**Comment by theDeemoonn** [comment]
Hello. Has a solution to the problem appeared?

**Comment by theDeemoonn** [comment]
@lionep Didn't find a solution to the problem?
